ci-operator/step-registry/ipi/conf/capability/baselinecaps/vset/ipi-conf-capability-baselinecaps-vset-commands.sh

```diff
@@ -63,9 +64,10 @@ v412=" ${v411} Console Insights Storage CSISnapshot"
 v413=" ${v412} NodeTuning"
 v414=" ${v413} Build DeploymentConfig ImageRegistry"
 v415=" ${v414} OperatorLifecycleManager CloudCredential"
-# shellcheck disable=SC2034
 v416=" ${v415} CloudControllerManager Ingress"
-latest_version="v416"
+# shellcheck disable=SC2034
+v417=" ${v416}"
+latest_version="v417"
```

Comment:
checked that capset `v4.17` is not supported yet on release 4.17 ( see [api](https://github.com/openshift/api/blob/master/config/v1/types_cluster_version.go#L431) as well)
```
$ ./openshift-install explain installconfig.capabilities.baselineCapabilitySet
KIND:     InstallConfig
VERSION:  v1

RESOURCE: <string>
  Valid Values: "None","v4.11","v4.12","v4.13","v4.14","v4.15","v4.16","vCurrent"
  baselineCapabilitySet selects an initial set of optional capabilities to enable, which can be extended via additionalEnabledCapabilities. The default is vCurrent.
```
This PR needs to wait for `v4.17` supported before merging.

Reply:
Yeah, understood, my 1st job failed because there is no v4.17 set yet.

Reply:
@jianlinliu capset `v4.18` is also supported on 4.18 ([OCPBUGS-41111](https://issues.redhat.com/browse/OCPBUGS-41111)), do we plan to update together in this PR or in separated PR?

Reply:
@jinyunma for v4.18, we do not need to consider optional operator yet, just add v4.18 baseline cap, right?

Reply:
I think so. From https://github.com/openshift/api/blob/master/config/v1/types_cluster_version.go#L589, no new optional operator is introduced or removed so far, same as v4.17.

I just know that capability `DeploymentConfig` is planed to be dropped from capset `vCurrent` and `v4.18` in 4.18, but didn't see code merged yet, maybe this can be updated later on demand.

Reply:
Okay, I will update v4.18 together in this PR, after all, v4.18 is indeed a new baseline cap. If new optional operator is introduced in the future, we can ask operator owner to update the list.